pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/IngestionConfig.java

```diff
@@ -43,15 +43,20 @@
   @JsonPropertyDescription("Configs related to record transformation functions applied during ingestion")
   private final List<TransformConfig> _transformConfigs;
 
+  @JsonPropertyDescription("Config related to handling complex type")
+  private final ComplexTypeHandlingConfig _complexTypeHandlingConfig;
```

Comment:
Thougths on renaming this to something simpler like "complexTypeConfig" or "flattenConfig"?

Reply:
+1 to `complexTypeConfig `